scaling_performance/optimizing_storage.adoc

```diff
@@ -13,6 +13,81 @@ toc::[]
 
 == Optimizing Storage
 
+== General Storage Guidance
+
+The following is a matrix which details the storage backend types {product-title} can be used with implementation examples and vendors.
+
+=== Available Storage Options
+[format="csv",width="70%",cols="3"]
+[options="header]
+|=======================
+Type,Examples,Vendors
+hostPath,"hostPath pod mounted volumes.  A specific type of Block Storage.  File paths on the node mounted onto running pods.",Others
+Block,"Block Storage. HDD, SSD, NVMe, Raw Physical Disk, VM Physical Disk, VMDK, LVM, iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, Dell/EMC Scale.IO, AWS EBS ^(1)^, GCE Persistent Disk ^(1)^, Azure Disk","RedHat, Seagate, WesternDigital, Toshiba, HGST, Intel, Samsung, OCZ, Plextor, PNY, Kingston, SanDisk, AWS, GCE, VMware, Dell/EMC, HPE, Others"
+File,"Network Attached File Storage.  CNS GlusterFS ^(1)^, GlusterFS ^(1)^, RHEL NFS, NetApp NFS ^(2)^, Vendor NFS, Vendor GlusterFS ^(3)^, Loopback Image over NFS","RedHat, NetApp, AWS, GCE, Others"
```

Comment:
@rootfs The guidance I received on CephFS was different.  Please see spreadsheet.   I see Azure File was added to https://docs.openshift.org/latest/architecture/additional_concepts/storage.html#types-of-persistent-volumes but OSP Manila was not.